janitor/functions.py

```diff
@@ -537,16 +539,12 @@ def fill_empty(df, columns, value):
         for col in columns:
             assert (
                 col in df.columns
-            ), "{col} missing from dataframe columns!".format(
-                col=col
-            )
+            ), "{col} missing from dataframe columns!".format(col=col)
```

Comment:
This, I believe, was the result of using `black` without the 79 character limit.

I am going to push up a `pyproject.toml` file, so that we can standardize on 79 characters when using the `black` tool.

Reply:
In the meantime, could you ensure that you use: `black -l 79` on the contents?